### ISSUE TITLE
Refactor fee usage for DIP20 tokens

### ISSUE THREAD

**Issue by riveign** [Psychedelic/plug#600]
## Description
DIP20 token transactions are failing when trying to send max amounts as the fee is not being properly handled. 

Right now fees are hard-coded for the default tokens (ICP, XTC, WICP) and some minor others (eg OGY).
- We should take the fee value from the token canister directly by serializing it into its metadata. 
      - https://github.com/Psychedelic/plug-mobile/pull/451/files
- We should refactor the front end to stop using hardcoded values and fetch the fee from the token.
    -  https://github.com/Psychedelic/plug-mobile/pull/441/files
    -  https://github.com/Psychedelic/plug-mobile/pull/442/files
    -  https://github.com/Psychedelic/plug-mobile/pull/448/files

TL;DR -> Actualizar controller a 0.20.3, eliminar la función getAssetFee y calcular el fee usando lo que viene del controller en cada token cuando haces getBalance
